src/Cpt.php

```diff
@@ -194,16 +196,18 @@ public function init() {
 	 * @since 0.1.0
 	 */
 	private function set_default_args() {
-		$this->set_args(array(
-			// The array of labels to use in the UI for this post type.
-			'labels' => $this->labels,
-			// Array of supported fields ( title, editor, thumbnail, etc ).
-			'supports' => $this->supports,
-			// We use the query var 'store' as opposed to the post type 'acf-store'.
-			'query_var' => strtolower( $this->singular ),
-			// Triggers the handling of re-writes for this post-type.
-			'rewrite' => $this->rewrite,
-		));
+		$this->set_args(
+			array(
```

Comment:
@honekun can we use the array short syntax here?

Reply:
Updated array syntax.
thanks @mannuel